OneCommunityGlobal/HighestGoodNetworkApp
# Yixiao header unittest

# Description
Create the unit test for the Header component to confirm whether it working as intended.

## Main changes explained:
- Create file Header.test.js to do the unit test.


## How to test:
1.check into current branch
2.do npm install to run this PR locally
3.run npm test src/components/Header/`__tests__`/Header.test.js

## Screenshots or videos of changes:
changed version:
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/49748716/4b613006-96a0-49e9-a11a-ba713ac26e57)
